### ISSUE TITLE
genindex directive is not working

### ISSUE THREAD

**Issue by danieltuzes** [sphinx-contrib/confluencebuilder#354]
I successfully created a documentation with Sphinx using the html output. Now I'd like to push it to Confluence and I have difficulties using the genindex directive. I am using [this SO workaround](<https://stackoverflow.com/questions/36235578/how-can-i-include-the-genindex-in-a-sphinx-toc>) to include the Index in the table of contents, so my \`index.rst\` looks like this:

```
Documentation
======================================
This is the documentation of ...

.. toctree::
   
   Documentation of my fancy stuff <self>
   installation
   innerFolder/index
   genindex
```

I have files `installation.rst` and `innerfolder/index.rst` , but I also have a `genindex.rst` in the `source` directory, where `genindex.rst` has the content:

```
Index
-------------------------------------
This file is a placeholder and will be replaced
```

If I create a html output using `sphinx-build -ab confluence source build/html` everything goes fine, the `build/html/genindex.html` will contain the proper index content, not the placeholder. But if I use the sphinxcontrib.confluencebuilder with `sphinx-build -ab confluence source build/html`, I don't any warning or error, but an index page with the placeholder text. It doesn't help if I add the directive to the `genindex.rst`, i.e. when the file content is

```
Index
-------------------------------------
This file is a placeholder and will be replaced

.. _genindex:
```

Any suggestion?

**Comment by jdknight** [comment]
At this time, this extension does not build any document-specific index (hence why `genindex` will remain as a placeholder). Will mark this as an enhancement request to see if this feature can be looked at in the future.

**Comment by danieltuzes** [comment]
Thanks @jdknight The description for the directive genindex clearly states what it should do. Not complying with the specification is rather a bug than a missing feature. At least confluencebuilder should warn the user that it cannot follow the specifications and genindex won't be created, and also, to mark this extension as an alpha version.

**Comment by jdknight** [comment]
Support for process `genindex` and other special pages has been added in #529; currently on the development branch and should be available next release (v1.7+). Please feel free to try it out by using the [bleeding edge sources](https://sphinxcontrib-confluencebuilder.readthedocs.io/en/stable/install/#development-installation) (if desired) and report any issues you may experience.

**Comment by jdknight** [comment]
v1.7 is now available on PyPI -- marking as closed.
